salt/salt.spec

```diff
@@ -1594,25 +1608,13 @@ rm -f %{_localstatedir}/cache/salt/minion/thin/version
 %dir               %attr(0750, root, root) %{_sysconfdir}/salt/minion.d/
 %dir               %attr(0750, root, root) %{_sysconfdir}/salt/pki/minion/
 %dir               %attr(0750, root, root) %{_localstatedir}/cache/salt/minion/
-%if %{with systemd}
-%{_sbindir}/rcsalt-minion
-%endif
-
 # Install plugin only on SUSE machines
 %if 0%{?suse_version}
 %{_prefix}/lib/zypp/plugins/commit/zyppnotify
 %endif
 
-# Install Yum plugins only on RH machines
-%if 0%{?fedora} || 0%{?rhel}
-%if 0%{?fedora} >= 22 || 0%{?rhel} >= 8
-%{python3_sitelib}/dnf-plugins/dnfnotify.py
-%{python3_sitelib}/dnf-plugins/__pycache__/dnfnotify.*
-%{_sysconfdir}/dnf/plugins/dnfnotify.conf
-%else
-%{_prefix}/share/yum-plugins/yumnotify.*
-%{_sysconfdir}/yum/pluginconf.d/yumnotify.conf
-%endif
+%if %{with systemd}
+%{_sbindir}/rcsalt-minion
```

Comment:
Looks like moved by accident, and not really related to this change, maybe better to move it back, just to make change smaller.